.github/workflows/cron-job-its.yml

```diff
@@ -17,14 +17,44 @@ name: Cron Job ITs
 on:
   schedule: # Runs by default on master branch
     - cron: '0 3 * * *' # Runs every day at 3:00 AM UTC
+  pull_request:
+    paths:
+      - 'owasp-dependency-check-suppressions.xml'
+    branches:
+      - master
+      - /^\d+\.\d+\.\d+(-\S*)?$/ # release branches
 
 jobs:
+  build:
+    if: github.event_name == 'schedule'
+    name: build (jdk8)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Setup java
+        run: export JAVA_HOME=$JAVA_HOME_8_X64
+
+      - name: Cache Maven m2 repository
+        id: maven
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: maven-${{ runner.os }}-8-${{ github.sha }}
+
+      - name: Maven build
+        id: maven_build
+        run: |
+          ./it.sh ci
```

Comment:
this works for old IT workflow as well?

Reply:
Yes, build happens the same way independently in the old (standard&revised) IT workflow. Also, old IT workflow builds docker container for revised ITs. Its not needed here as of now.